source/docs/software/advanced-controls/introduction/tuning-flywheel.rst

```diff
@@ -0,0 +1,238 @@
+
+   Despite the differences in capitalization, the two formats refer to the same concept.
+
+
+In this section, we'll go through some techniques to manually find reasonable values for the gains in a PID controller.
+
+This is useful if you are not using the :ref:`SysId toolsuite <docs/software/pathplanning/system-identification/index:System Identification>`. Additionally, even if you are using it, it is useful to see and understand the behavior of changing the values of the constants in different situations.
+
+Working with PID Controllers
+----------------------------
+
+Use this interactive simulation to explore tuning concepts:
+
+.. raw:: html
+
+    <div class="viz-div">
+      <div id="flywheel_pid_container">
+         <div class="col" id="flywheel_pid_plotVals"></div>
+         <div class="col" id="flywheel_pid_plotVolts"></div>
+      </div>
+      <div class="flex-grid">
+         <div class="col" id="flywheel_pid_viz"></div>
+         <div id="flywheel_pid_ctrls"></div>
+      </div>
+      <script>
+         flywheel_pid = new FlywheelPIDF("flywheel_pid");
+         flywheel_pid.runSim();
+      </script>
+    </div>
+
+Flywheel Tuning Step 1: Feedback-Only
+-------------------------------------
+
+We will first attempt to tune the flywheel using only the feedback terms :math:`K_p`, :math:`K_i`, and :math:`K_d`.
+
+Perform the following:
+
+1. Set :math:`K_p`, :math:`K_i`, :math:`K_d`, and :math:`K_v` to zero.
+2. Increase :math:`K_p` until the :term:`output` starts to oscillate around the :term:`setpoint`.
+3. Increase :math:`K_d` as much as possible without introducing jittering in the :term:`system response`.
+4. *In some cases*, increase :math:`K_i` if :term:`output` gets "stuck" before converging to the :term:`setpoint`.
+
+.. important:: Adding an integral gain to the :term:`controller` is often a sub-optimal way to eliminate :term:`steady-state error`. As we will see soon, a better approach is to incorporate feedforward.
+
+.. note:: When "increasing" a value, multiply it by two until the expected effect is observed. Similarly, when "decreasing" a value, divide by two. Once you find the point where the expected effect starts or stops, switch to "bumping" the value up and down by ~10% until the behavior is good enough.
+
+.. raw:: html
+
+   <details>
+     <summary>Tuning Solution</summary><br>
+
+
+In this particular example, for a setpoint of 1000, values of :math:`K_p = 0.13`, :math:`K_i = 0.0`, and :math:`K_d = 0.002` will produce somewhat reasonable results. It will get better or worse as you change the setpoint.
+
+.. raw:: html
+
+   </details> <br>
+
+
+Flywheel Tuning Step 2: Feedforward, then Feedback
+--------------------------------------------------
+
+Tuning with only feedback can produce reasonable results in many cases. However, there is an easier way. Rather than starting with feedback, start by calibrating an appropriate feedforward value.
```

Comment:
Per above, "many cases" should be more specific.  It works well in systems where no control signal is required to maintain steady state at the setpoint, and poorly elsewhere.

Reply:
Maybe give examples (drive/turret position vs elevators and velocity systems)